packages/editor/src/lib/editor/types/external-content.ts

```diff
@@ -24,7 +23,7 @@ export type TLExternalContentSource =
 	  }
 
 /** @public */
-export type TLExternalContent = {
+export type TLExternalContent<EmbedDefinition> = {
```

Comment:
Made this generic so `Editor.ts` does not need to know about the concrete `EmbedDefinition` type.